packages/CodingStandard/src/Fixer/Property/ArrayPropertyDefaultValueFixer.php

```diff
@@ -70,8 +70,13 @@ private function isArrayPropertyDocComment(Token $token): bool
         return true;
     }
 
-    private function addDefaultValueForArrayProperty(Token $semicolonToken): void
+    private function addDefaultValueForArrayProperty(Tokens $tokens,int $semicolonPosition): void
     {
-        $semicolonToken->setContent(' = [];');
+        $tokens->insertAt($semicolonPosition, new Token(']'));
+        $tokens->insertAt($semicolonPosition, new Token('['));
+        $tokens->insertAt($semicolonPosition, new Token(' '));
```

Comment:
sadly, that's how internal PHP's parser works

feel free to propose how to handle it in nicer way

waiting for travis for new issue it will reveal